src/odbc/winlogin.c

```diff
@@ -173,5 +173,9 @@ LoginDlgProc(HWND hDlg, UINT message, WPARAM wParam,	/* */
 bool
 get_login_info(HWND hwndParent, TDSLOGIN * login)
 {
+#ifdef DLL_EXPORT
```

Comment:
Why this is `DLL_EXPORT` and not, for instance `TDS_NO_DM` ?

Reply:
Okay, I got it, if you are not a DLL you don't have the dialog to load. But then all code above is not used too.